guides/ansible_engine/4-runplaybook/README.md

```diff
@@ -0,0 +1,37 @@
+# Exercise 4 - Running the apache-basic-playbook
+
+Congratulations!  You just wrote a playbook that incorporates some key Ansible concepts that you use in most if not all of your future playbooks.  Before you get too excited though, we should probably make sure it actually runs.
```

Comment:
@gdykeman why was this made a separate exercise in the guide? Seems it should be merged with 3. Don't why the would be separated like this.

Reply:
+1 - I would very much like to merge them. Exercise 4 is just the execution, nothing else.